File: app/src/main/java/com/example/manajemenuser/pojo/ResponseGetAllOffline.java
package com.example.manajemenuser.pojo;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Awesome Pojo Generator
 * */
public class ResponseGetAllOffline {

  @SerializedName("name")
  @Expose
  private String name;
  @SerializedName("id")
  @Expose
  private Integer id;
  @SerializedName("email")
  @Expose
  private String email;
  private String date;

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    private int status;

    public ResponseGetAllOffline(String name, String email, String date,int status) {
        this.date =date;
        this.name = name;
        this.email = email;
        this.status = status;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

  public void setName(String name){
   this.name=name;
  }
  public String getName(){
   return name;
  }
  public void setId(Integer id){
   this.id=id;
  }
  public Integer getId(){
   return id;
  }
  public void setEmail(String email){
   this.email=email;
  }
  public String getEmail(){
   return email;
  }
}

File: app/src/main/java/com/example/manajemenuser/pojo/ResponseGetAllOffline.kt
package com.example.manajemenuser.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Awesome Pojo Generator
 */
class ResponseGetAllOffline(
    @field:Expose @field:SerializedName("name") var name: String,
    @field:Expose @field:SerializedName("email") var email: String,
    var date: String,
    var status: Int
) {

    @SerializedName("id")
    @Expose
    var id: Int? = null

}